src/components/undo-redo-buttons/undo-redo-buttons.tsx

```diff
@@ -18,17 +21,21 @@ export const UndoRedoButtons = ({
   isUndoDisabled,
   isRedoDisabled,
 }: IUndoRedoButtons): ReactElement => (
-  <div className="flex items-center">
+  <div className={cn(style.undoRedo)}>
     <UndoRedoButton onClick={onUndo} disabled={isUndoDisabled}>
       <Icon name="Undo" />
     </UndoRedoButton>
 
-    <UndoRedoButton onClick={onRedo} disabled={isRedoDisabled} className="ml-2">
+    <UndoRedoButton
+      onClick={onRedo}
+      disabled={isRedoDisabled}
+      className={style.undoRedo_button_right}
+    >
       <Icon name="Redo" />
     </UndoRedoButton>
 
-    <Link className="flex flex-wrap ml-3" to={Routes.Root}>
-      <Icon size={15} name="Close" className="text-white cursor-pointer" />
+    <Link className={cn(style.undoRedo_closeIcon)} to={Routes.Root}>
```

Comment:
I think that close link is not a logical part of UndoRedo buttons, can we extract it

Reply:
Done